src/js/wrappers/ListFilterFormWrapper.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+const ListFilterFormWrapper = ({ children, className }) => (
```

Comment:
There are 6 wrappers that are basically the same differing with only "default" classes that are added to the className:
1. `d-flex flex-column list-page-filters ${className}`
2. `d-flex justify-content-end buttons align-items-center ${className}`
3. `d-flex list-page-header ${className}`
4. `title-text p-3 d-flex justify-content-between align-items-center ${className}`
5. `list-page-list-section ${className}`
6. `d-flex flex-column list-page-main ${className}`

Do you really need all of them?

Reply:
I agree at the moment it could be one, that would just receive className prop, but I think it is more maintainable with this approach, especially if we'd want to add any element in the specific wrapper and if we want to use it also while creating the forms.